1/README.md

```diff
@@ -3,23 +3,12 @@
 To execute:
 
 * Install npm
-* Install vue cli: "npm install -g @vue/cli"
-* Run "npm install" in the VueJS Folder
-* Run "npm run serve' in the VueJS Folder
+* Install Neo4j Community server and run on Port 7687
+* Username and Password should both be "neo4j" (Or change in the neo4j.js file)
+* Create a User in Neo4j to login
```

Comment:
Good that you add documentation how to setup your code. What do I need to do exactly to create the user?

Reply:
Added pagination with SKIP and LIMIT parameters to the todos query